Assets/FPSCapper.prefab

```diff
@@ -0,0 +1,45 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &9169474616986226046
```

Comment:
Besides that we don't need this prefab, also avoid placing prefabs inside the root folder